### PR TITLE
Refactor FXIOS-7301 - Remove 1 closure_body_length violation from FxAPushMessageHandler.swift

### PR DESCRIPTION
**Pull request by ionixjunior**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-7301)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/16442)

## :bulb: Description

This PR removes 1 `closure_body_length` violation from the `FxAPushMessageHandler.swift` file by extracting object creation into separate functions. Specifically, it refactors the creation of the push message error and the creation of the push message by device command.

This PR is part of a series of PRs described here https://github.com/mozilla-mobile/firefox-ios/issues/16442#issuecomment-2197676804.

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

File: firefox-ios/Account/FxAPushMessageHandler.swift [hunk 1]
```diff
@@ -83,6 +63,35 @@ extension FxAPushMessageHandler {
             }
         }
     }
+
+    private func getPushMessageErrorBy(result: Result<AccountEvent, Error>) -> PushMessageError {
```
Comment (FilippoZazzeroni) [inline firefox-ios/Account/FxAPushMessageHandler.swift:67]: Nit: i'd do `getPushErrorMessageFrom(result:)` or `makePushErrorMessageFrom(result:)`

File: firefox-ios/Account/FxAPushMessageHandler.swift [hunk 2]
```diff
@@ -83,6 +63,35 @@ extension FxAPushMessageHandler {
...
+        } else {
+            self.logger.log("Got zero events from FxA",
+                            level: .warning,
+                            category: .sync,
+                            description: "No events retrieved from fxa")
+            return PushMessageError.messageIncomplete("empty message")
+        }
+    }
+
+    private func getPushMessageBy(deviceCommand: IncomingDeviceCommand) -> PushMessage {
```
Comment (FilippoZazzeroni) [inline firefox-ios/Account/FxAPushMessageHandler.swift:83]: Same here `makePushMessageFrom(deviceCommand:)` or `getPushMessageFrom(deviceCommand:)`

**Review by FilippoZazzeroni — APPROVED** [review]
Nit things. Great job again 🎉

**Comment by ionixjunior** [comment]
Thank you! Updated and rebased.

**Pull request merged**